packages/public/inspection-capture-web/src/PhotoCaptureHUD/PhotoCaptureHUDSightPreview/SightsSlider/SightsSlider.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, IconName } from '@monkvision/common-ui-web';
+import { Sight } from '@monkvision/types';
+import { useSightLabel } from '@monkvision/common';
+import { labels } from '@monkvision/sights';
+import { RefObject, useEffect, useRef } from 'react';
+import { styles } from './SightsSlider.styles';
+
+export interface SightsSliderProps {
+  sights: Sight[];
+  sightSelected: Sight;
+  sightsTaken: Sight[];
+  onSightSelected?: (sight: Sight) => void;
+}
+
+interface SliderButtonProps {
+  sight: Sight;
+  sightSelected: Sight;
+  sightsTaken: Sight[];
+  onSightSelected: (sight: Sight) => void;
+  label: (sight: Sight) => string;
+}
+
+function SliderButton({
+  sight,
+  sightSelected,
+  sightsTaken,
+  onSightSelected,
+  label,
+}: SliderButtonProps) {
+  const isSelected = sight.id === sightSelected.id;
+  const isTaken = sightsTaken.some((sightTaken) => sightTaken.id === sight.id);
+
+  let primaryColor = 'secondary-xdark';
+  let icon = undefined as IconName | undefined;
+
+  if (isSelected) {
+    primaryColor = 'primary-base';
+  }
+  if (isTaken) {
+    primaryColor = 'primary-light';
+    icon = 'check';
+  }
+  return (
+    <Button
+      style={styles['button']}
+      icon={icon}
+      primaryColor={primaryColor}
+      onClick={() => {
+        onSightSelected(sight);
+      }}
+      data-testid={`sight-btn-${sight.id}`}
+    >
+      {label(sight)}
+    </Button>
+  );
+}
+
+const onScrollToSelected = (
```

Comment:
From what I understand, this function **actually** scrolls to the selected sight, so you can rename it to `scrollToSelectedChild` or something like that